### PR TITLE
Add new clauses to the Cypher manual glossary.

### PR DESCRIPTION
**Pull request by Lojjs**
Added the following clauses to the glossary:
- existential subqueries
- CALL subqueries
- USE (fabric only)

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/clause.asciidoc [hunk 1]
```diff
@@ -59,6 +59,7 @@ These comprise sub-clauses that must operate as part of reading clauses.
 |===
 |Sub-clause                                         | Description
 |<<query-where,WHERE>>                          | Adds constraints to the patterns in a `MATCH` or `OPTIONAL MATCH` clause or filters the results of a `WITH` clause.
+|<<existential-subqueries, WHERE EXISTS {}>>    | An existential sub-query used to filter the results of a `MATCH`, `OPTIONAL MATCH` or `WITH` clause.
```
Comment (SaschaPeukert) [inline cypher/cypher-docs/src/docs/dev/clause.asciidoc]:
```suggestion
|<<existential-subqueries, WHERE EXISTS { ... }>>    | An existential sub-query used to filter the results of a `MATCH`, `OPTIONAL MATCH` or `WITH` clause.
```

File: cypher/cypher-docs/src/docs/dev/clause.asciidoc [hunk 2]
```diff
@@ -129,7 +130,7 @@ These comprise clauses that both read data from and write data to the database.
 [options="header"]
 |===
 |Clause                             |   Description
-|<<query-call-subquery,CALL {} (subquery)>>  | Evaluates a subquery returning values.
+|<<query-call-subquery,CALL {} (subquery)>>  | Evaluates a subquery, typically used for post-union processing or aggregations.
```
Comment (SaschaPeukert) [inline cypher/cypher-docs/src/docs/dev/clause.asciidoc]: Don't you think `CALL {} (subquery)` looks weird compared to the others?

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc [hunk 3]
```diff
@@ -21,7 +21,8 @@ This section comprises a glossary of all the keywords -- grouped by category and
 [options="header"]
 |===
 |Clause                                     | Category      |   Description
-|<<query-call, CALL [...YIELD]>>                        | Reading/Writing   | Invoke a procedure deployed in the database.
+|<<query-call, CALL procedure>>             | Reading/Writing   | Invoke a procedure deployed in the database.
```
Comment (SaschaPeukert) [inline cypher/cypher-docs/src/docs/dev/glossary.asciidoc]: don't we like yield anymore?

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc [hunk 4]
```diff
@@ -21,7 +21,8 @@ This section comprises a glossary of all the keywords -- grouped by category and
...
+|<<query-call-subquery, CALL {} (subquery)>> | Reading/Writing   | Evaluates a subquery, typically used for post-union processing or aggregations.
```
Comment (SaschaPeukert) [inline cypher/cypher-docs/src/docs/dev/glossary.asciidoc]: same question as everywhere else this is used

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc [hunk 5]
```diff
@@ -49,13 +50,15 @@ This section comprises a glossary of all the keywords -- grouped by category and
 |<<query-union, UNION>>                      | Set operations   |  Combines the result of multiple queries. Duplicates are removed.
 |<<query-union, UNION ALL>>                      | Set operations   |  Combines the result of multiple queries. Duplicates are retained.
 |<<query-unwind, UNWIND ... [AS]>>                    | Projecting   |  Expands a list into a sequence of rows.
+|<<query-use, USE>>                           | Multiple graphs | [fabric]#Determines which graph a query, or query part, is executed against.#
```
Comment (SaschaPeukert) [inline cypher/cypher-docs/src/docs/dev/glossary.asciidoc:53]: What are the #s for?

**Review by SaschaPeukert — CHANGES_REQUESTED** [review]

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc (hunk 5 above)
Comment (Lojjs) [reply cypher/cypher-docs/src/docs/dev/glossary.asciidoc:53]: Magic to get fabric box.

**Review by SaschaPeukert — APPROVED** [review]

**Pull request merged**